### ISSUE TITLE
Maven central does not have protoc prebuilts

### ISSUE THREAD

**Issue by yujincheng08** [protocolbuffers/protobuf#13070]
**What version of protobuf and what language are you using?**
3.23.3 for Java

**What operating system (Linux, Windows, ...) and version?**
Linux

**What runtime / compiler are you using (e.g., python version or gcc version)**
Java 17

**What did you do?**
1. Go to https://repo.maven.apache.org/maven2/com/google/protobuf/protoc/3.23.3/
3. Scroll down and notice there's no `protoc-3.23.3-linux-x86_64.exe` or so.

**What did you expect to see**
There's protoc prebuilts like previous version, e.g. https://repo.maven.apache.org/maven2/com/google/protobuf/protoc/3.23.2/

**What did you see instead?**

No protoc prebuilts

**Anything else we should know about your project / environment**
I use gradle protoc plugin and here you are the error message:
```
Execution failed for task ':app:generateDebugProto'.
> Could not resolve all files for configuration ':app:protobufToolsLocator_protoc'.
   > Could not find protoc-3.23.3-linux-x86_64.exe (com.google.protobuf:protoc:3.23.3).
     Searched in the following locations:
         https://repo.maven.apache.org/maven2/com/google/protobuf/protoc/3.23.3/protoc-3.23.3-linux-x86_64.exe
```

**Comment by deannagarcia** [comment]
Thanks for the report and sorry for the inconvenience, I have just pressed the release button for the protoc binaries (sometimes this can take a day or so to roll out)

**Comment by drogozov-da** [comment]
Does it have something to do with wrong CRC's in https://repo1.maven.org/maven2/com/google/protobuf/protoc/3.23.2/ ?

**Comment by drogozov-da** [comment]
Posted a new issue about checksum mismatch here https://github.com/protocolbuffers/protobuf/issues/13082

**Comment by uuhnaut69** [comment]
any updates?

**Comment by yujincheng08** [comment]
AFAIK, no one can modify a released version from maven central. So the only thing we can do is to wait for the next release.

**Comment by lfarkas** [comment]
this is still not propagated....

**Comment by yu-shiba** [comment]
@deannagarcia There seems to be a time lag in the release of protoc binaries each time, where is the problem?
If the problem is with the CDN, we have to wait, but if it is in the release cycle, can we hold off on releasing them to the maven repository until we have the full set of deliverables?

**Comment by albertoalegria** [comment]
Any other solutions besides downgrading?

**Comment by Kurru** [comment]
Any update?

**Comment by ingoschaal** [comment]
Hi,
we have the same problem for a build pipeline we download the files but with this version it does not work,
can you upload this to maven central? This version or a new release?
thank you for the support

https://repo.maven.apache.org/maven2/com/google/protobuf/protoc/3.23.3/protoc-3.23.3-windows-x86_64.exe
https://repo.maven.apache.org/maven2/com/google/protobuf/protoc/3.23.3/protoc-3.23.3-linux-x86_64.exe

   > Could not find protoc-3.23.3-linux-x86_64.exe (com.google.protobuf:protoc:3.23.3).
     Searched in the following locations:
         https://repo.maven.apache.org/maven2/com/google/protobuf/protoc/3.23.3/protoc-3.23.3-linux-x86_64.exe

**Comment by elliott-miller** [comment]
We are also unable to upgrade due to this. In our case, we are using the maven plugin: https://www.xolstice.org/protobuf-maven-plugin/

**Comment by zhangskz** [comment]
We attempted to reupload the protoc prebuilts, but per https://central.sonatype.org/faq/can-i-change-a-component/, unfortunately it looks like we can't update 23.3 with the missing artifacts after the fact.

We are planning to release 23.4 soon (tentatively next week, pending some additional fixes). We recommended waiting to upgrade to the next patch, or downgrading in the meantime. Apologies for the inconvenience!
